services/spar/src/Spar/SCIM.hs

```diff
@@ -399,6 +399,10 @@ createScimToken :: Maybe UserId -> CreateScimToken -> Spar ScimToken
 createScimToken zusr CreateScimToken{..} = do
     let descr = createScimTokenDescription
     teamid <- getZUsrOwnedTeam zusr
+    tokenNumber <- fmap length $ wrapMonadClient $ Data.getScimTokens teamid
+    maxTokens <- asks (maxScimTokens . sparCtxOpts)
+    unless (tokenNumber < maxTokens) $
```

Comment:
Ah!  This should really be `<=`, I think.  (See my comment on the related test case.)

Reply:
I think it should be `<`. If we already have `maxTokens` tokens, we can't add a new one. So we should fail unless there's space for at least one more token.

Reply:
right.  my mistake, sorry.